lib/src/pages/messages.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:flutter/material.dart';
+
+class Messages extends StatefulWidget {
+  const Messages({Key? key}) : super(key: key);
+
+  @override
+  _MessagesState createState() => _MessagesState();
+}
+
+class _MessagesState extends State<Messages> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.teal.shade800,
```

Comment:
Make a file in constants -> app_colors.dart. Declare the color there and access it here